makerepropkg.in

```diff
@@ -153,8 +153,7 @@ for fname in "${installed[@]}"; do
         exit 1
     fi
 done
-printf '%s\n' "${allpkgfiles[@]}" | mkarchroot -U "${archroot_args[@]}" "${buildroot}"/root - || exit 1
-
+printf '%s\n' "${allpkgfiles[@]}" | mkarchroot -M @pkgdatadir@/makepkg-x86_64.conf -U "${archroot_args[@]}" "${buildroot}"/root - || exit 1
```

Comment:
Okay, I suppose this is not unreasonable. Though it is the *guest* makepkg.conf which is the issue here, not the host. mkarchroot only replaces the makepkg.conf installed by installing pacman using pacstrap, *iff* there is an `-M` argument.